app/views/applications/show.html.erb

```diff
@@ -21,19 +21,14 @@
 
       <%= t.body do %>
         <% @k8s_data.each do |environment, data| %>
-          <% not_on_default_branch = capture do %>
```

Comment:
Oh wait - you've now removed this section? So can drop the previous two commits?